tests/test_data.py

```diff
@@ -50,21 +50,33 @@ def trajectory_rew(trajectory: types.Trajectory) -> types.TrajectoryWithRew:
 
 
 @pytest.fixture
-def transitions(
+def transitions_min(
     obs_space: gym.Space, act_space: gym.Space, length: int
+) -> types.TransitionsMinimal:
+    obs = np.array([obs_space.sample() for _ in range(length)])
+    acts = np.array([act_space.sample() for _ in range(length)])
+    infos = np.array([{}] * length)
```

Comment:
Again, if this is optional, we should have a test case for that.

Reply:
Not optional